### ISSUE TITLE
Add Italian translation.

### ISSUE THREAD

**Issue by boromyr** [Qalculate/qalculate-gtk#504]
Here are the files of my translation into Italian, partially revised.
[qalculate-gtk.zip](https://github.com/Qalculate/qalculate-gtk/files/13547326/qalculate-gtk.zip)

**Comment by hanna-kn** [comment]
I noticed the text string that states that it is an automatic translation (using GPT-3). To what extent have you checked that the translation is correct?

Problems I've noted in the translation (at a quick glance):
- The translation of capitalized text strings is highly inconsistent (I assume that capitalization should generally not be used in the Italian translation)
- Inconsistent translation of bitwise AND, OR, and NOT ("AND bit a livello di bit", "OR Bitwise", "Operatore NOT bitwise")
- "Copy Full Text" translated as "Translate the following text into Italian..."
- "[EXPRESSION]" translated as "Regressione agli antropomorfi"

**Comment by boromyr** [comment]
Yes, I used an automatic translation engine and therefore there are many inaccuracies, I have corrected quite a few, but there are others that I am fixing. There are many, and it is not easy to find them all. I will update the files after further review.

**Comment by boromyr** [comment]
The strings that contain "_" what function do they have?

**Comment by hanna-kn** [comment]
> The strings that contain "_" what function do they have?

The character after "_" is underlined and used as mnemonic accelerator key.

**Comment by boromyr** [comment]
Ok, that's fine, wait until I finish completing other corrections and the strings with "_" and I will upload a beta version more suitable for publication. The remaining corrections can be reported by the community in this chat.

**Comment by boromyr** [comment]
[LC_MESSAGES.zip](https://github.com/Qalculate/qalculate-gtk/files/13693905/LC_MESSAGES.zip)

**Comment by Efois** [comment]
> Ok, that's fine, wait until I finish completing other corrections and the strings with "_" and I will upload a beta version more suitable for publication. The remaining corrections can be reported by the community in this chat.

Do you need hep in reviewing them?
